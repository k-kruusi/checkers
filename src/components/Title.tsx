import { theme } from '../theme';
import '../theme/fonts.css';
import { getString } from '../utils';

const titleStyle: React.CSSProperties = {
  textAlign: 'center',
  color: theme.colors.ivory,
  fontSize: '4em',
  fontFamily: theme.fonts.amatic,
};

export const Title = () => {
  return <div style={titleStyle}><h1>{getString("company")} {getString("title")}</h1></div>;
}

export const Checkers = () => {
  return <div style={titleStyle}><h1>{getString("title")}</h1></div>;
}

export const Birdseye = () => {
  return <div style={titleStyle}><h1>{getString("company")}</h1></div>;
}