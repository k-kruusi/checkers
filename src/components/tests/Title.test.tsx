import { render } from '@testing-library/react';
import { Title } from '../Title';

describe('Title', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(<Title />);

    expect(asFragment()).toMatchSnapshot();
  });
});