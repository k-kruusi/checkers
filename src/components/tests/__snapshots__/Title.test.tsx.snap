// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title matches the snapshot 1`] = `
<DocumentFragment>
  <div
    style="text-align: center; color: rgb(255, 255, 240); font-size: 4em; font-family: Amatic SC, sans-serif;"
  >
    <h1>
      Birdseye Checkers
    </h1>
  </div>
</DocumentFragment>
`;
